scripts/release/index.js

```diff
@@ -16,14 +13,33 @@ cooker.run('publish', [
   cook.relatedPackagesByPackage,
   cook.getCurrentVersionByPackage,
   cook.evaluateNewVersionByPackage,
+  cook.byBranch,
```

Comment:
We base all the @next versions on "latest" so this works correctly: we take latest "master release", compute semver and then apply semver on top of latest "master release version". Good for me.